src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const descriptionTransfers = description.split('\n');
+  const notificationBlock = `
+    <div class="notification ${type}" style="
+      position: absolute; 
+      top: ${posTop}px; 
+      right: ${posRight}px;
+    ">
+      <h2>${title}</h2>
+      <p>${descriptionTransfers[0]}
+      <br>${descriptionTransfers[1]}
+      </br>
+    </div>
+  `;
+
+  body.insertAdjacentHTML('afterend', notificationBlock);
+
+  setTimeout(() => [...document.querySelectorAll('.notification')]
+    .forEach(elem => elem.remove()),
+  2000);
```

Comment:
don't remove all elements, remove only exact one you've created in this function